conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -283,28 +284,6 @@ public void resumeTracking() {
         convivaVideoAnalytics.reportPlaybackEvent(event);
         Log.d(TAG, "Tracking resumed.");
     }
-
-    /**
-     * This should be called when the app is resumed
-     */
-    public void reportAppForegrounded() {
-        Log.d(TAG, "appForegrounded");
-        ConvivaAnalytics.reportAppForegrounded();
-        isBackgrounded = false;
-    }
-
-    /**
-     * This should be called when the app is paused
-     */
-    public void reportAppBackgrounded() {
-        Log.d(TAG, "appBackgrounded");
-        if (!isBackgrounded) {
-            ConvivaAnalytics.reportAppBackgrounded();
-            isBackgrounded = true;
-        }
-
-    }
-
```

Comment:
Do you know if these calls still exist for other SDK's (especially react native)? If so let's make sure we align across SDK's.
Also worth a changelog entry I think.

Reply:
I reverted those blocks, as I noticed that `ConvivaVideoAnalytics` and `ConvivaAdAnalytics` provides public constructors (in contrast to `ConvivaAnalytics.buildVideoAnalytics` and `ConvivaAnalytics.buildAdAnalytics`) , which are not registering to the `ActivityLifecycleCallbacks`. I added some javadoc to let the consumer know about this functions: https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/78/commits/d7a2a00ce065a0643505c6ccfe12775cbd2b2681

Reply:
👍  In the future we might decide to remove them but we should probably do that with deprecation for a while.